code/game/machinery/recycler.dm

```diff
@@ -148,10 +148,9 @@ var/const/SAFETY_COOLDOWN = 100
 	else
 		playsound(src.loc, 'sound/effects/splat.ogg', 50, 1)
 
-	var/gib = 1
-	// By default, the emagged recycler will gib all non-carbons. (human simple animal mobs don't count)
+	// By default, the emagged recycler will gib both carbons and non-carbons. (human simple animal mobs don't count)
 	if(iscarbon(L))
-		gib = 0
+		L.gib()
 		if(L.stat == CONSCIOUS)
 			L.say("ARRRRRRRRRRRGH!!!")
 		add_blood(L)
```

Comment:
Why would you add blood to a body thats about to be gibbed?? Just spawn a blood decal below!